logstash.go

```diff
@@ -10,20 +10,21 @@ import (
 // Hook represents a connection to a Logstash instance
 type Hook struct {
 	conn net.Conn
+	appName string
 }
 
 // NewHook creates a new hook to a Logstash instance, which listens on
 // `protocol`://`address`.
-func NewHook(protocol, address string) (*Hook, error) {
+func NewHook(protocol, address string, appName string) (*Hook, error) {
```

Comment:
a small nitpick: change `NewHook(protocol, address **string**, appName string)` to `NewHook(protocol, address, appName string)`